1/debian-10/rootfs/opt/bitnami/scripts/libmediawiki.sh

```diff
@@ -66,6 +66,7 @@ mediawiki_validate() {
         for empty_env_var in "MEDIAWIKI_SMTP_USER" "MEDIAWIKI_SMTP_PASSWORD"; do
             is_empty_value "${!empty_env_var}" && warn "The ${empty_env_var} environment variable is empty or not set."
         done
+        is_empty_value "$MEDIAWIKI_SMTP_AUTH" && warn "The MEDIAWIKI_SMTP_AUTH environment variable is empty or not set."
```

Comment:
Since it is a boolean value, instead of including its raw value, we could try this:

```suggestion
        check_yes_no_value "MEDIAWIKI_ENABLE_SMTP_AUTH"
```

Reply:
The expected value is boolean. But, **check_yes_no_value** is trying to enforce yes/no. Are you suggesting that MEDIAWIKI_ENABLE_SMTP_AUTH should be yes/no, causing us to convert to boolean when writing the settings file? 

Are we trying to enforce that this always be defined?

Reply:
Yes, I think it makes sense to add a default value (`no`). Note that `check_yes_no_value` not only allows for `yes`/`no` but also `true`/`false`.

Reply:
I see, but the error would print that only yes/no are valid:

```
check_yes_no_value() {
    if ! is_yes_no_value "${!1}" && ! is_true_false_value "${!1}"; then
        print_validation_error "The allowed values for ${1} are: yes no"
    fi
}
```

Anyway, the latest changes should work...

Reply:
Should we add a default value in Dockerfile?

```
...
MEDIAWIKI_ENABLE_SMTP_AUTH="no" \
...
```